Soccar [UI]/src/nl/socnet/message/MovePlayerMessage.java

```diff
@@ -0,0 +1,51 @@
+package nl.socnet.message;
+
+import nl.soccar.library.enumeration.HandbrakeAction;
+import nl.soccar.library.enumeration.SteerAction;
+import nl.soccar.library.enumeration.ThrottleAction;
+import nl.soccar.socnet.message.Message;
+import nl.soccar.socnet.message.MessageConstants;
+import nl.soccar.socnet.message.MessageEvent;
+import nl.socnet.message.handler.MovePlayerMessageHandler;
+
+/**
+ *
+ * @author PTS34A
+ */
+@MessageEvent(id = MessageConstants.MOVE_PLAYER_MESSAGE_ID, handler = MovePlayerMessageHandler.class)
+public final class MovePlayerMessage extends Message {
+
+    private final String username;
+    private final SteerAction steerAction;
+    private final HandbrakeAction handbrakeAction;
+    private final ThrottleAction throttleAction;
+
+    public MovePlayerMessage(String username, SteerAction steerAction, HandbrakeAction handbrakeAction, ThrottleAction throttleAction) {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public constructor. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)